modules/managed-cluster-add-notification-contacts.adoc

```diff
@@ -12,8 +12,8 @@ By default, only the cluster owner receives cluster notification emails. You can
 
 .Prerequisites
 * Your cluster is deployed and registered to the {hybrid-console}.
-* You are logged in to the {hybrid-console-second}.
-* The intended notification recipient has a user account on your cluster.
+* You are logged in to the {hybrid-console-second} as the cluster owner or as a user with the cluster editor role.
+* The intended notification recipient has a Red Hat Customer Portal account associated with the same organization as the cluster owner.
```

Comment:
Nit - per the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces) there should be a non-breaking space between Red and Hat:
```suggestion
* The intended notification recipient has a Red{nbsp}Hat Customer Portal account associated with the same organization as the cluster owner.
```